### PR TITLE
feat :  done with product listing

### PR DESCRIPTION
**Pull request by iprankurpandey**

### CODE REVIEW

File: product.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    align-items: center;
+    flex-direction: row;
+    height: 2rem;
+    background-color: silver;
+    max-width: 100%;
+    font-size: 1.5rem;
+}
+
+.fa {
+    margin: 0.5rem;
```
Comment (omkarpatke) [inline product.css:42]: if the value is less than 1rem then try to use px.

File: product.css [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    display: flex;
+}
+
+.mtrlshop {
+    font-size: 2rem;
+}
+
+.catagory-items-page {
+    display: grid;
+    grid-template-columns: 250px auto;
```
Comment (omkarpatke) [inline product.css:55]: use rem here I don't have an exact reason here but someone says it's a good practice.

**Review by omkarpatke — CHANGES_REQUESTED** [review]

**Comment by amantulsyan35** [comment]
apart from the changes @omkarpatke suggested, looks good to me

File: product.html [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+                        <div class="btn-icons-vcard">
+                            <button class="btn-add-to-vcart btn-class-vcard">cart<span
+                                    class="material-icons">shopping_cart</span></button>
+                            <button class="btn-add-to-vwishlist btn-class-vcard"><span
+                                    class="material-icons">favorite_border</span></button>
+                        </div>
+                    </div>
+                </div>
+
+
```
Comment (Adarsh-Patel-Dev) [inline product.html:144]: remove extra spaces here

File: product.html [hunk 4]
```diff
@@ -0,0 +1,330 @@
...
+        <div class="social-media-links">
+            <div class="github"><i class="fa fa-github"></i></div>
+            <div class="twitter"><i class="fa fa-twitter"></i></div>
+            <div class="Linkedin"><i class="fa fa-linkedin"></i></div>
+        </div>
+        <div class="H5">© 2021 Agri UI </div>
+    </footer>
+
+</body>
+
```
Comment (Adarsh-Patel-Dev) [inline product.html:329]: Apart from above suggestions Everything is fine
It can be merged

**Review by Adarsh-Patel-Dev — APPROVED** [review]

**Review by mehulsatardekar — APPROVED** [review]
Yep agree with @omkarpatke's suggestions. other than that branch is good to merge.

File: product.html [hunk 5]
```diff
@@ -0,0 +1,330 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product Listing</title>
+    <link rel="stylesheet" href="/product.css">
+</head>
+
+<body>
+
+
+    <div class="navigation-menu">
```
Comment (Adarsh-Patel-Dev) [inline product.html:15]: use nav tag here

**Review by Adarsh-Patel-Dev — CHANGES_REQUESTED** [review]

**Pull request merged**